.github/workflows/Dependency-Security-Audit.yml

```
@@ -0,0 +1,26 @@
+  schedule:
+    - cron: '0 0 * * 0'  # Runs weekly
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Dependencies
+      run: npm install
+    - name: Run npm audit
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2002:style:3:5: Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead

(shellcheck)